src/quicktype-core/index.ts

```diff
@@ -14,8 +14,15 @@ export {
     InferenceFlagName,
     RunContext
 } from "./Run";
-export { CompressedJSON } from "./input/CompressedJSON";
-export { Input, InputData, JSONInput, JSONSourceData, jsonInputForTargetLanguage } from "./input/Inputs";
+export { CompressedJSON, CompressedJSONFromString } from "./input/CompressedJSON";
+export {
+    Input,
+    InputData,
+    JSONInput,
+    JSONSourceData,
+    jsonInputForTargetLanguage,
+    json5InputForTargetLanguage
```

Comment:
Where is json5 support more appropriate? A new pkg `quicktype-json5-input` or `quicktype-core`?